src/features/comments/commentsSlice.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Instead of disabling eslint for this import, consider reorganizing your imports to avoid circular dependencies.